applications/adv_networking_bench/adv_networking_bench_default_tx_rx.yaml

```diff
@@ -104,15 +98,6 @@ advanced_network:
               udp_src: 4096 #12288
               udp_dst: 4096 #12288
               ipv4_len: 1050
-          - name: "ADC Samples2"
-            id: 1
-            action:
-              type: queue
-              id: 1
-            match:
-              udp_src: 4095 #12288
-              udp_dst: 4095 #12288
-              ipv4_len: 1050              
```

Comment:
Could you please help explain this change? Why remove?

Reply:
done in #611 @tbirdso